### ISSUE TITLE
newrelic "online" buildpack fails to resolve agent 8.30.0.0 due to regex, apps fail to start

### ISSUE THREAD

**Issue by aegershman** [newrelic/newrelic-dotnet-buildpack-tile#5]
**TL;DR** the lookup pattern matching for the newrelic buildpack to download the latest newrelic agent is 404ing 😉 

---

We're using 1.1.5 of the [newrelic-dotnet-buildpack](https://network.pivotal.io/products/new-relic-dotnet-buildpack/); when using the "online" (e.g. the _not_ `cached` version) buildpack, where the buildpack reaches out to `download.newrelic.com` to get the version of the newrelic agent to use, apps are failing during staging due to the newrelic-dotnot buildpack getting 404's:

We turned on debugging and saw the agent was reaching out to:

>http://download.newrelic.com/dot_net_agent/previous_releases/8.30.0.0/SHA256/newrelic-netcore20-agent_8.30.0.0_amd64.tar.gz.sha256

which doesn't exist. However, this does:

>http://download.newrelic.com/dot_net_agent/previous_releases/8.30.0/SHA256/newrelic-netcore20-agent_8.30.0.0_amd64.tar.gz.sha256

In `/.../newrelic-dotnetcore-extension/.../supply.go` [it appears a regex is used to consume the latest available newrelic agent](https://github.com/newrelic/newrelic-dotnet-buildpack-tile/blob/master/core-extension/src/newrelic-dotnetcore-extension/supply/supply.go#L84-L87) listed on `download.newrelic.com/...` under `previous_releases` [as seen here](http://download.newrelic.com/dot_net_agent/previous_releases). 

As of writing this, `8.30.0` is the latest agent listed in that `previous_releases` index; however, the regex the newrelic dotnet buildpack appears to be searching for is `8.30.0.0`, which doesn't exist in the `previous_releases` (though `8.30.0` does), thus the buildpack `supply` throwing 404's

We've switched to using the `cached` (`nr_dotnetcore_extension_cached `) buildpack for now, but anyone who is using the "online" buildpack variant (`nr_dotnetcore_extension`) would have problems when restaging their dotnet applications unless they pinned the url, version, or until a newrelic agent version of 8.30.0.0 or an `x.x.x.x` of the newrelic agent were published 👍 

thanks for your time all

**Comment by shahramk** [comment]
@no problem @aegershman - I noticed that you were using version 1.1.5 from May of 2020. There are newer releases after that. This issue was addressed in later releases and is fixed. Please use the latest release from March 2021.
